AIO/Techdebt.md

```diff
@@ -0,0 +1,2 @@
+# Tech Debt
+- exchangeContext is deprecated
```

Comment:
Can we remove this file?

Although `exchangeContext` is "deprecated" in the OSBot API, it isn't truly deprecated, it's marked for internal use, but there is no reason that script developers cannot user it.

Reply:
Oh okay good. I asked in chat but no one had a good answer. Do you ever use the chat?

Reply:
Not so much recently